src/hint_cli.py

```diff
@@ -1,19 +1,21 @@
 """
 Usage:
-  ./hint start [--pull] [<configname>]
+  ./hint start [--pull] [--hintr-branch=<branch>] [--hint-branch=<branch>] [<configname>]
```

Comment:
do we need the angles on this? I thought that `[--hintr-branch=branch]` would work? Please check if you make the change as docopt can be weird

Reply:
From [docopt docs](http://docopt.org/) most examples use `--option=<value>`. It does say 

> Follow either <angular-brackets> or UPPER-CASE convention for options' arguments.